cluster/userdata-master.yaml

```diff
@@ -1063,6 +1063,17 @@ write_files:
       parameters:
         type: gp2
 
+  - path: /srv/kubernetes/manifests/thirdpartyresources/platform-i-a-m-credentials.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: ThirdPartyResource
+      metadata:
+        # NOTE: dashes are needed to get proper PlatformIAMCredentials name
+        name: platform-i-a-m-credentials.zalando.org
```

Comment:
My gut feeling is that we should stick with only a simple namespace like `zalando.org` as everything else opens discussions on when to open a new namespace and what granularity a namespace has to have. The name `PlatformIAMCredentials` was suggested by @sarnowski and I'm OK with it but I would be also open for alternatives --- nobody else raised any issues with the name `PlatformIAMCredentials` yet, so right now I would stay with it.